password-hash/src/salt.rs

```diff
@@ -118,25 +118,37 @@ impl<'a> Salt<'a> {
             err => err,
         })
     }
+    /// use `from_b64`
+    #[deprecated(since = "0.5.0", note = "the name was unclear, use `from_b64` instead")]
+    pub fn new(input: &'a str) -> Result<Self> {
+        Self::from_b64(input)
+    }
 
     /// Attempt to decode a B64-encoded [`Salt`], writing the decoded result
     /// into the provided buffer, and returning a slice of the buffer
     /// containing the decoded result on success.
     ///
     /// [1]: https://github.com/P-H-C/phc-string-format/blob/master/phc-sf-spec.md#argon2-encoding
-    pub fn b64_decode<'b>(&self, buf: &'b mut [u8]) -> Result<&'b [u8]> {
+    pub fn to_raw<'b>(&self, buf: &'b mut [u8]) -> Result<&'b [u8]> {
         self.0.b64_decode(buf)
     }
+    /// use `to_raw`
+    #[deprecated(since = "0.5.0", note = "the name was unclear, use `to_raw` instead")]
+    pub fn b64_decode<'b>(&self, buf: &'b mut [u8]) -> Result<&'b [u8]> {
+        self.to_raw(buf)
+    }
 
     /// Borrow this value as a `str`.
     pub fn as_str(&self) -> &'a str {
         self.0.as_str()
     }
 
+    /*
     /// Borrow this value as bytes.
     pub fn as_bytes(&self) -> &'a [u8] {
         self.as_str().as_bytes()
     }
+    */
```

Comment:
I removed this method because it is misleading in the rust context. Since this would leak a not ideal memory representation